generator/tests.py

```diff
@@ -228,8 +236,22 @@ def test_render_checklist_affects_prerelease(self):
         )
         for release in ("5.1 before 5.1.8", "5.0 before 5.0.14"):
             with self.subTest(release=release):
-                expected = f"[row 1] Django\n      [row 2] {release}"
-                self.assertIn(expected, checklist_content)
+                expected = f"[row 1] Django [row 2] {release}"
+                self.assertInChecklistContent(expected, checklist_content)
+
+        cves = checklist.securityissue_set.all()
+        prenotification = [
+            "Create a new text file `prenotification-email.txt` with content",
+            "a set of security releases will be issued on Wednesday, May 7, 2025 "
+            "around 1118 UTC",
```

Comment:
I think maybe the time format is not quite right?

Reply:
Yeah, I saw that, but it matches our current format: since we usually use "top of the hour" times, this looks like: "around 1400 UTC" which is "correct" (military time).

Having said that, I think we could improve this and use ":" between hour and minutes. I will push this change, but let me know if you disagree.

Reply:
Ah good point - we can leave it :+1: